cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js

```diff
@@ -0,0 +1,496 @@
+/**
+ * @file cypress/tests/integration/Z_ArticleViewDCMetadata.spec.js
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ */
+
+describe('Article View Metadata - DC Plugin', function() {
+	let submission;
+	let uniqueId;
+	let today;
+	let dcElements;
+
+	before(function() {
+		today = new Date();
+		const uniqueSeed = Date.now().toString();
+		uniqueId = Cypress._.uniqueId(uniqueSeed);
+
+		submission = {
+			section: 'Articles',
+			prefix: 'Test prefix',
+			title: 'Test title',
+			subtitle: 'Test subtitle',
+			abstract: 'Test abstract',
+			authors: [
+				'Name 1 Author 1', 
+				'Name 2 Author 2'
+			],
+			submitterRole: 'Journal manager',
+			additionalAuthors: [
+				{
+					'givenName': 'Name 1',
+					'familyName': 'Author 1',
+					'country': 'United States',
+					'affiliation': 'Stanford University',
+					'email': 'nameauthor1Test@mailinator.com',
+				},
+				{
+					'givenName': 'Name 2',
+					'familyName': 'Author 2',
+					'country': 'United States',
+					'affiliation': 'Stanford University',
+					'email': 'nameauthor2Test@mailinator.com',
+				}
+			],
+			localeTitles: {
+				fr_CA: {
+					title: "Test title FR",
+					subtitle: "Test subtitle FR",
+					abstract: "Test abstract FR",
+					prefix: "Test prefix FR",
+				}
+			},
+			localeMetadata: [
+				{
+					locale: 'fr_CA',
+					manyValues: [
+						{
+							metadata: 'keywords',
+							values: [
+								'Test keyword 1 FR',
+								'Test keyword 2 FR'
+							],
+						},
+					],
+					oneValue: [
+						{
+							metadata: 'coverage',
+							value: 'Test coverage FR'
+						},
+						{
+							metadata: 'type',
+							value: 'Test type FR'
+						},
+					],
+				},
+				{
+					locale: 'en_US',
+					manyValues: [
+						keywords = {
+							metadata: 'keywords',
+							values: [
+								'Test keyword 1',
+								'Test keyword 2'
+							],
+						},
+					],
+					oneValue: [
+						{
+							metadata: 'coverage',
+							value: 'Test coverage'
+						},
+						{
+							metadata: 'type',
+							value: 'Test type'
+						},
+					],
+				}
+			],
+			source: {
+				name: 'Journal of Public Knowledge',
+				issn: '0378-5955',
+				issue: '2',
+				volume: '1',
+				uri: 'http://localhost/index.php/publicknowledge',
+				issueTitle: 'Vol. 1 No. 2 (2014)',
+				doiPrefix: '10.1234',
+			},
+			identifiers: {
+				pageNumber: '71-98',
+				uri: '',
+				doi: '',
+			},
+			urlPath: 'testing-dc-metadata-submission-' + uniqueId,
+			licenceUrl: 'https://www.google.com',
+			publishIssueSections: [
+				'Articles'
+			],
+			galleys: [
+				{
+					label: 'PDF',
+					genre: 'Article Text',
+					mimeType: 'application/pdf',
+				}
+			],
+		};
+
+		dcElements = {
+			localised: [
```

Comment:
I think we should use 'localized' -- the American English version :-)

Reply:
Fixed.